x-pack/plugins/observability/public/pages/slos/components/slo_list_item.tsx

```diff
@@ -200,6 +223,21 @@ export function SloListItem({
       {isDeleteConfirmationModalOpen ? (
         <SloDeleteConfirmationModal slo={slo} onCancel={handleDeleteCancel} />
       ) : null}
+
+      {isAddRuleFlyoutOpen ? (
+        <AddRuleFlyout
+          consumer={ALERTS_FEATURE_ID}
```

Comment:
Consumer must be sloFeatureId exported from `x-pack/plugins/observability/common/index.ts`